Dockerfile

```
@@ -1,14 +1,17 @@
-FROM haskell:9.4.5
-RUN mkdir -p /app/user
+FROM haskell:9.4.5 as BIN
 WORKDIR /app/user
 COPY *.cabal ./
 
 RUN cabal v2-update
 RUN cabal v2-build --dependencies-only
 
-COPY . /app/user
-RUN cabal v2-install
+COPY . .
+RUN cabal v2-install --install-method copy --installdir . --overwrite-policy=always
 
+FROM debian:12.5-slim
+RUN apt update
+RUN apt install -y ca-certificates
```

Om vi kör hadolint så kommer den klaga på att vi inte rensar apt cachen efter vi installerat våra paket

---

Samt att vi inte specar exakta versioner, men det klarar vi oss utan

---

Spelar inte så stor roll för det bara är i build stage

---

Hadolint rekommenderar detta: 

```suggestion
RUN apt-get update && apt-get install --no-install-recommends -y ca-certificates \
 && apt-get clean \
 && rm -rf /var/lib/apt/lists/*
```

https://github.com/hadolint/hadolint/wiki/DL3009

---

> Spelar inte så stor roll för det bara är i build stage

Nope, det är i den slutgiltiga imagen väl?